### PR TITLE
added storage values to api-server

### PR DESCRIPTION
**Pull request by jasmingacic**
@fog1985 I messed up the previous PR so I reopened it here.
Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

### CODE REVIEW

File: charts/api-server/templates/deployment.yaml [hunk 1]
```diff
@@ -38,6 +38,21 @@ spec:
...
+          - name: "STORAGE_SECRETACCESSKEY"
+            value: "{{ .Values.storage.accessKey }}"
+          - name: "STORAGE_LOCATION"
+            value: "{{ .Values.storage.location }}"
+          - name: "STORAGE_TOKEN"
+            value: "{{ .Values.storage.token }}"
+          - name: "STORAGE_SSL"
+            value:  "{{ .Values.storage.SSL }}"
+          - name: "SCRAPPERENABLED"
+            value:  "{{ .Values.storage.scrapperDisabled }}"
```
Comment (exu) [inline charts/api-server/templates/deployment.yaml:54]: Enabled or disabled 🤩

Comment (nicufk) [reply charts/api-server/templates/deployment.yaml:54]: 😆

Comment (jasmingacic) [reply charts/api-server/templates/deployment.yaml]: duh :D

**Review by nicufk — APPROVED** [review]
👍

**Pull request merged**